app/postinstallscript.sh

```diff
@@ -39,16 +39,9 @@ EOF
 # reload daemon for service file changes to take effect
 systemctl daemon-reload
 
-# Create docker symbolic link
+# Create docker and cli-plugins directory
 mkdir -p /usr/local/bin
-ln -s /usr/local/packages/dockerdwrapperwithcompose/docker /usr/local/bin/docker
-
-# Create docker-compose symbolic link
 mkdir -p /usr/local/lib/docker/cli-plugins
```

Comment:
remove this - the acap user will not be able to create these directories and neither should it be done here